src/Sulu/Component/Content/Exception/InvalidNavigationContextExtension.php

```diff
@@ -41,11 +41,11 @@ public function __construct($selectedNavContexts, $navContexts)
     }
 
     /**
-     * @return \string[]
+     * @return string[]
      */
     public function getSelectedNavContext()
     {
-        return $this->selectedNavContext;
+        return $this->selectedNavContexts;
```

Comment:
is set as `selectedNavContexts` in the constructor. 
maybe I shoule deprecate `getSelectedNavContext` and add `getSelectedNavContexts`?